src/assets/custom/css/_sass/_curated-publications.scss

```diff
@@ -1,19 +1,35 @@
 .curated-publications {
-  background-color: rgba(0, 255, 0, 0.1);
-}
-
-.curated-publications--intro {
-  background-color: rgba(0, 255, 0, 0.1);
+  background: $sheen;
 }
 
 .curated-publications--cards {
-  background-color: rgba(0, 255, 0, 0.1);
+  @include medium-large-and-extra-large {
+    margin-bottom: -152px;
+  }
 }
 
-.curated-publications--card {
-  background-color: rgba(0, 255, 0, 0.1);
+.curated-publications--cta-space {
+  @include lateral-spacing;
+  text-align: center;
+
+  @include small {
+    padding-bottom: 50px;
+  }
+  @include medium-large-and-extra-large {
+    background-color: #1e2631;
+    padding-top: 224px;
+    padding-bottom: 122px;
+  }
 }
 
 .curated-publications--cta {
-  background-color: rgba(0, 255, 0, 0.1);
+  margin-bottom: 28px;
+  margin-top: 28px;
+
```

Comment:
Round up Paddings/Margins to nearest 0s & 5s

Reply:
Done in this commit: https://github.com/codurance/site/pull/1725/commits/c2ec0e5c09ebbcb2e0c8acf44dd4124fa2cf044f